src/main/java/seedu/address/model/room/RoomList.java

```diff
@@ -225,13 +225,14 @@ public void setRooms(PriorityQueue<Room> rooms) {
 
     /**
      * Checks if the given room number is present in the application.
+     *
      * @param roomNumber to check if it is in the application.
      * @return Index Of room that is found.
      */
-    public Index checkIfRoomPresent(Integer roomNumber) {
+    public static Index checkIfRoomPresent(Integer roomNumber, ObservableList<Room> roomList) {
```

Comment:
I am not sure if it is a good idea to make this function static and expose it to the outer world, considering how you also need to call the getRoomList() function from the model to get the ObservableList<Room> as your parameter for this method so the roomList will be from your existing RoomList class anyway  :/. Just a thought to consider!

Reply:
This is one of the dilemma I'm facing. Whether to have a static method in RoomList.java or have a method in EditRoomCommand.java (that is out of place). I think i will adopt the same method of just using the observablelist.stream.anyMatch() method and not create a new method. How does it sound?

Reply:
i will shift this method to the model manager also